modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library_settings/preview_limit.jsp

```diff
@@ -0,0 +1,102 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/document_library/init.jsp" %>
+
+<%
+DLPreviewLimitConfigurationDisplayContext dlPreviewLimitConfigurationDisplayContext = (DLPreviewLimitConfigurationDisplayContext)request.getAttribute(DLPreviewLimitConfigurationDisplayContext.class.getName());
+%>
+
+<aui:form action="<%= dlPreviewLimitConfigurationDisplayContext.getEditDLPreviewLimitConfigurationURL() %>" method="post" name="fm">
+	<clay:sheet>
+		<liferay-ui:error exception="<%= ConfigurationModelListenerException.class %>" message="there-was-an-unknown-error" />
+
+		<liferay-ui:error exception="<%= DLPreviewLimitConfigurationException.InvalidMaxFileSizeException.class %>">
+			<liferay-ui:message key="maximum-file-size-limit-is-not-valid" />
+		</liferay-ui:error>
+
+		<liferay-ui:error exception="<%= DLPreviewLimitConfigurationException.InvalidMaxNumberOfPagesException.class %>">
+			<liferay-ui:message key="maximum-number-of-pages-limit-is-not-valid" />
+		</liferay-ui:error>
+
+		<clay:sheet-header>
+			<h2 class="c-mb-4">
+				<liferay-ui:message key="dl-preview-limit-configuration-name" />
+			</h2>
+		</clay:sheet-header>
+
+		<clay:sheet-section>
+			<h3 class="c-mb-2 sheet-subtitle text-2 text-secondary"><liferay-ui:message key="dl-preview-limit-configuration-maximum-file-size-title" /></h3>
```

Comment:
Remove` text-2` class, it is setting the font-size to 12px and should be 14px (the default for `.sheet-subtitle`)

Reply:
Now I see that you may have copied this classes from the File Upload Limit section, and there we also have `text-2` class. It is wrong also there. Would you mind also removing from there? 
We should try to avoid to override clay css styles, but also in this case, in the Figma the size is the one that Clay sets.

Reply:
Done